iree/vm/list.c

```diff
@@ -505,8 +505,9 @@ IREE_API_EXPORT void* iree_vm_list_get_ref_deref(
   return value.ptr;
 }
 
-IREE_API_EXPORT iree_status_t iree_vm_list_get_ref_assign(
-    const iree_vm_list_t* list, iree_host_size_t i, iree_vm_ref_t* out_value) {
+static iree_status_t iree_vm_list_get_ref_assign_or_retain(
```

Comment:
For us mortals, would you mind adding a comment to this function describing what it does?